lib/Compiler.js

```diff
@@ -321,7 +321,7 @@ Compiler.prototype.emitAssets = function(compilation, callback) {
 				}
 				var content = source.source();
 				if(!Buffer.isBuffer(content))
```

Comment:
Maybe `"curly": ["error", "multi-line", "consistent"]`, to still allow for short `if`s that cannot visually be confused with a block?